src/utilities/hypre_memory.c

```diff
@@ -1248,9 +1248,13 @@ HYPRE_Int
 hypre_umpire_device_pooled_allocate(void **ptr, size_t nbytes)
 {
    hypre_Handle *handle = hypre_handle();
+   //const hypre_int device_id = hypre_HandleCudaDevice(handle);
```

Comment:
@Ram3sh @davidbeckingsale I tried to add Umpire support following instructions from @Ram3sh with `DEVICE::X` where `X` is device id (see commented code here), but got segfault. Any ideas of what I am doing wrong? Thanks!

Reply:
What version of Umpire are you using?

Reply:
@davidbeckingsale

I use the `develop` branch up to this commit.

```
commit 6ba5e4dd9756cb0cbc4a204443bc96c075d587d3 (HEAD -> develop, origin/develop, origin/HEAD)
Merge: 0054f38f 8180fabd
Author: David Beckingsale <davidbeckingsale@gmail.com>
Date:   Tue Jan 12 16:01:24 2021 -0800

    Merge pull request #521 from LLNL/bugfix/UM-815-fix-CMakeLists
    
    removing extra print statement in CMakeLists
```

Reply:
Hi, @davidbeckingsale, is this problem easy to fix? If not, we may leave it as is and merge the other parts of this PR in. Just asking. Thanks! -Ruipeng

Reply:
Everything looks fine - can you email me more details about the segfault?